src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskListNotesModal.tsx

```diff
@@ -0,0 +1,318 @@
+              </Typography>
+            </Grid>
+            <Grid item xs={12} md={12}>
+              <Typography fontWeight={'bold'}>Notes:</Typography>
+              <Box sx={{ height: '200px', overflow: 'auto' }}>
+                <Typography> {task.notes}</Typography>
+              </Box>
+            </Grid>
+          </Grid>
+        </DialogContent>
+      </Dialog>
+    );
+  };
+
+  const EditModal: React.FC = () => {
+    return (
+      <Dialog fullWidth maxWidth={dialogWidth} open={modalShow} onClose={onHide}>
+        <DialogTitle className={'font-weight-bold'}>{task.title}</DialogTitle>
+        <DialogContent>
+          <form
+            id={'create-work-package-form'}
+            onSubmit={(e) => {
+              e.preventDefault();
+              e.stopPropagation();
+              handleSubmit(onSubmit)(e);
+            }}
+            onKeyPress={(e) => {
+              e.key === 'Enter' && e.preventDefault();
+            }}
+          >
+            <Grid container spacing={2}>
+              <Grid item xs={12} md={7}>
+                <FormControl sx={{ width: '100%' }}>
+                  <FormLabel>Title</FormLabel>
+                  <Controller
+                    name={'title'}
+                    control={control}
+                    rules={{ required: true }}
+                    render={({ field: { onChange, value } }) => (
+                      <TextField
+                        required
+                        onChange={onChange}
+                        value={value}
+                        inputProps={{
+                          maxlength: isUnderWordCount(value, 15) ? null : 0
+                        }}
+                        helperText={`${countWords(value)}/15 words`}
+                        error={!isUnderWordCount(value, 15)}
+                      />
+                    )}
+                  />
+                </FormControl>
+              </Grid>
+              <Grid item xs={12} md={5}>
+                <FormControl fullWidth>
+                  <FormLabel>Priority</FormLabel>
+                  <Controller
+                    name="priority"
+                    control={control}
+                    render={({ field: { onChange, value } }) => (
+                      <TextField select onChange={onChange} value={value} error={!!errors.priority}>
+                        {Object.values(TaskPriority).map((p) => (
+                          <MenuItem key={p} value={p}>
+                            {p}
+                          </MenuItem>
+                        ))}
+                      </TextField>
+                    )}
+                  />
+                </FormControl>
+              </Grid>
+              <Grid item xs={12} md={8}>
+                <FormControl fullWidth>
+                  <FormLabel>Assignees</FormLabel>
+                  <Controller
```

Comment:
can we use the autocomplete thing we have pre made?

Reply:
the ner autocomplete? no because this one is a multiselect that ones just for one thing I believe.